simplematrixbotlib/match.py

```diff
@@ -82,17 +82,27 @@ def __init__(self, room, event, bot, prefix="") -> None:
         """
         super().__init__(room, event, bot)
         self._prefix = prefix
-        bot_user = self.room.users[self.room.own_user_id]
-        # in case another user uses the same display name, we need to regard the disambiguation if present
-        self._disambiguated_name = bot_user.disambiguated_name
-        # still we allow to mention without disambiguation if typing manually for now
-        self._display_name = bot_user.display_name
-        # element generates a "pill" from formatted (HTML) links to matrix.to/#/@matrix:id
-        # this isn't really specced but still the status quo
-        self._pill = f'<a href="https://matrix.to/#/{self.room.own_user_id}">'
-        self._body_without_prefix = None
-
-    def command(self, command=None):
+
+        """Forms of identification"""
+        self._own_user_id = f"@{self._bot.creds.username}:{self._bot.creds.homeserver.replace("https://","").replace("http://","")}"
```

Comment:
I think this does not work for homeservers with delegation:
the homeserver variable is `matrix.example.org` but the id is `@bot:example.org`